bloom/generators/debian/templates/ament_cmake/copyright.em

```diff
@@ -1 +1,11 @@
-@(Copyright)
\ No newline at end of file
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: @(Name)
+Upstream-Contact: @(Maintainers.replace(', ', '\n '))
+@[if Source and Source != '']Source: @(Source)@\n@[end if]@
+@[for License, Text in Licenses]@
+
+Files: *
+Copyright: See package copyright in source code for details
+License: @(License)
+@[if Text and Text != ''] @(Text)@\n@[end if]@
```

Comment:
Is it possible that in the case of missing `Text` here we end up generating a non compliant copyright file since the license text is needed?

Reply:
I've added some placeholder text referring the consumer to the repository itself to find the full text.